patroni/dcs.py

```diff
@@ -240,8 +240,11 @@ def take_leader(self):
         overwriting the key if necessary."""
 
     @abc.abstractmethod
-    def initialize(self):
+    def initialize(self, create_new=True, sysid=None):
```

Comment:
Does it make seance to set default value of sysid argument to an empty string?
This will help to get rid from following unnecessary checks:
1) sysid or "" in the etcd.py
2) sysid if sysid else "" in the zokeeper.py
3) sysid.encode("utf-8") if sysid else ""  in the zokeeper.py